dashboards/google-kubernetes-engine-enterprise/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+sample_dashboards:
+  -
+    category: Google Kubernetes Engine
+    id: gke-enterprise-cluster-observability-overview
+    display_name: Google Kubernetes Engine (GKE) Enterprise Cluster Observability Overview
```

Comment:
Can this be "GKE Enterprise Cluster Observability Overview" to match the title in the json?

Reply:
Sure - Done!